### DISCUSSION TITLE
Urgenza recupero password

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#171] [Richieste di aiuto]
Buongiorno, abbiamo provato in tutti i modi possibili ed immaginabili ,ma la configurazione delle mail non funziona ,di conseguenza anche i docenti che vogliono recuperare la password magari persa non riescono e devono per forza chiamare in segreteria

**Comment by trinko** [comment]
Potrebbe anche essere un problema di Altervista.
Comunque nella prossima versione c'è un apposita configurazione con mail di test iniziale e appositi messaggi di errore. Dovrebbe risolvere

**Comment by sempliciniluigi89** [comment]
La mia scuola non utilizza Altervista ,bensì x10hosting .Aspetto con ansia l'uscita della prossima versione
